packages/relay/src/lib/clients/cache/ICacheClient.ts

```diff
@@ -22,12 +22,13 @@ export interface ICacheClient {
   get(key: string, callingMethod: string, requestIdPrefix?: string): any;
   set(key: string, value: any, callingMethod: string, ttl?: number, requestIdPrefix?: string): void;
```

Comment:
```suggestion
  set(key: string, value: any, callingMethod: string, ttl?: number, requestIdPrefix?: string): Promise<void> | void;
```

Reply:
Well, it makes sense for both implementations to be using Promises, if they are going to follow a common interface. We can add async modifier to the `LocalLRUCache` methods and use Promises as return types in the interface. In this way both classes will conform to the interface and it wouldn't have any effect on the actual implementation.